.github/workflows/publish-chart.yaml

```diff
@@ -4,6 +4,11 @@ name: Publish Helm chart
 on:
   workflow_dispatch:
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
```

Comment:
just to make sure I understand correctly - `closed` is needed to check `github.event.pull_request.merged`?
https://github.com/aquasecurity/trivy/blob/ea1ba74603772aca11e1b9b853c1e7e4717be2aa/.github/workflows/publish-chart.yaml#L79

Reply:
> just to make sure I understand correctly - `closed` is needed to check `github.event.pull_request.merged`?
> 
> https://github.com/aquasecurity/trivy/blob/ea1ba74603772aca11e1b9b853c1e7e4717be2aa/.github/workflows/publish-chart.yaml#L79
yes, I keep the default types and add `closed` for PR merged